nixos/modules/services/torrent/cross-seed.nix

```diff
@@ -138,9 +170,11 @@ in
             "use strict";
             const fs = require("fs");
             const settings_json = "${jsonSettingsFile}";
-            let loaded_settings = JSON.parse(fs.readFileSync(settings_json, "utf8"));
+            let loaded_settings = {};
+            ${genConfigSegment}
+            Object.assign(loaded_settings, JSON.parse(fs.readFileSync(settings_json, "utf8")));
             ${secretSettingsSegment}
-            module.exports = loaded_settings;
+            module.exports = settings;
```

Comment:
Shouldn't this still equal `loaded_settings`? Sorry, I'm on mobile so I might be missing something, but it looks like `settings` was never defined.

Reply:
Oops, I'd renamed it, then decided to not rename it and forgot that line. Thanks

Reply:
Done.